src/Compilers/Core/Portable/Emit/AnonymousTypeValue.cs

```diff
@@ -2,28 +2,14 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
-
-using Microsoft.Cci;
 using System.Diagnostics;
 
-namespace Microsoft.CodeAnalysis.Emit
-{
-    [DebuggerDisplay("{Name, nq}")]
-    internal readonly struct AnonymousTypeValue
-    {
-        public readonly string Name;
-        public readonly int UniqueIndex;
-        public readonly ITypeDefinition Type;
-
-        public AnonymousTypeValue(string name, int uniqueIndex, ITypeDefinition type)
-        {
-            Debug.Assert(!string.IsNullOrEmpty(name));
-            Debug.Assert(uniqueIndex >= 0);
```

Comment:
Is there a reason to remove these checks? Also, are any changes needed to this file? If not, consider reverting.